### ISSUE TITLE
Create more user friendly functions

### ISSUE THREAD

**Issue by areibman** [AgentOps-AI/tokencost#5]
The tokencost module isn't intuitive. Currently there are a few exported functions:
`calculate_cost`, `count_message_tokens`, `count_string_tokens`, and `get_max_completion_tokens`.

1. `calculate_cost` makes no sense. It would be clearer if it were a general string tokenizer+cost calculator instead of a number cruncher. Right now, calculate_cost just multiples numbers by their cost lookups. It should be updated so that it looks like:
a) calculate_cost(messages, model=model) -> 12345
b) calculate_cost(string, model=model) -> 12345

The doc string also has the wrong types.

Alternatively, we create a new function that does act as a general string tokenizer+cost calculator and keep calculate cost as it currently is.

2. `count_message_tokens` is an extension on top of tiktoken that just adds some extra padding tokens. This is helpful, and we should keep it. `count_string_tokens` is more or less the same.

3. `get_max_completion_tokens` doesn't have a great name and/or doc string. It reads: `Calculate the maximum number of completion tokens for a given model and list of messages.` This doesn't communicate what it's doing well. This function takes a list of messages, and given the context window of the model, it says how many tokens you have left. In other words, if the model has an 8k token limit and 1k tokens are used in the messages arg, then it should return 7k.

**TL;DR**
Let's work backwards. Write a README.md of an example of Tokencost that works in a way that you as a user would want it to work. Then, we can rename/refactor our exported functions to fit that. Here's my thinking:

```
from tokencost import calculate_cost

calculate_cost('hello world', model='gpt-3.5-turbo')
>>>5

messages = [
{'role': 'user': 'hello world'},
{'role': 'assistant': 'how may I help you?'}
]

calculate_cost(messages=messages, model='gpt-4')
>>>12

```
